tests/unit.py

```diff
@@ -0,0 +1,71 @@
+# This file contains a series of unit tests that compare expected and actual javascript code generated by pyd3xter
+# instructions:
+# install pytest: http://pytest.org/latest/
+# $> python -m pytest -v unit.py
+# TODO: refactor to streamline js comparisons; consider edge cases, syntactic curbballs
+# note: expected_js field hand-written
```

Comment:
Rather than making this note, you can convert `expected_js` to `EXPECTED_JS`. The all-caps is convention for hard-coding.